src/base/utils/misc.cpp

```diff
@@ -356,14 +356,22 @@ QString Utils::Misc::friendlyUnit(qint64 bytesValue, bool isSpeed)
         return QCoreApplication::translate("misc", "Unknown", "Unknown (size)");
     QString ret;
     if (unit == SizeUnit::Byte)
-        ret = QString::number(bytesValue) + " " + unitString(unit);
+        ret = QString::number(bytesValue) + QString::fromUtf8(C_NON_BREAKING_SPACE) + unitString(unit);
     else
-        ret = Utils::String::fromDouble(friendlyVal, 1) + " " + unitString(unit);
+        ret = Utils::String::fromDouble(friendlyVal, friendlyUnitPrecision(unit)) + QString::fromUtf8(C_NON_BREAKING_SPACE) + unitString(unit);
     if (isSpeed)
         ret += QCoreApplication::translate("misc", "/s", "per second");
     return ret;
 }
 
+int Utils::Misc::friendlyUnitPrecision(const SizeUnit &unit)
+{
+    // friendlyUnit's number of digits after the decimal point
```

Comment:
I put the comment there as an explanation for returning magic numbers. I don't see how it'll be useful near the declaration as IMO ~~the name is pretty self explanatory~~
i'll be damned, i thought scale is precision (just looked it up) should i change name to `friendlyUnitScale` or `friendlyUnitAccuracy`?